assets/prior_versions_index.sql

```diff
@@ -0,0 +1,77 @@
+-- Step 1: Declare variables
+DECLARE idc_versions ARRAY<INT64>;
+DECLARE latest_idc_version INT64;
+DECLARE union_all_query STRING;
+
+-- Step 2: Get all idc_versions
+SET idc_versions = (
+  SELECT [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18]
```

Comment:
```suggestion
  SELECT GENERATE_ARRAY(1, latest_idc_version)
  -- SELECT [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18]
```

Reply:
this is only temporary and will be updated dynamically with commented out query once Bill updates the `bigquery-public-data.idc_current.version_metadata` metadata table